extras/rya.export/README.md

```diff
@@ -0,0 +1,250 @@
+**Child:** Rya repository that was copied from parent then modified.  
+**older, newer:** Statement timestamp is older/newer than specified merge time (start time).  
+**missing:** Statement is absent from the repository.  
+
+| **case** | **Parent** | **Child** | **assume that** | **merge modification** | **in parent after** |
+|----------|------------|-----------|-----------------|------------------------|---------------------|
+| 1        | older      | missing   | child deleted   | delete from parent     | no                  |
+| 2        | newer      | missing   | parent added    | do nothing             | yes                 |
+| 3        | missing    | older     | parent deleted  | do nothing             | no                  |
+| 4        | missing    | newer     | child added     | add to parent          | yes                 |
+| 5x       | older      | older     | same key        | do nothing             | yes                 |
+| 6x       | newer      | newer     | same key        | do nothing             | yes                 |
+| 7\*      | older      | newer     | child updated   | do cases 1 and 4       | yes                 |
+| 8\*      | newer      | older     | parent updated  | do cases 2 and 3       | yes                 |
+
+NOTE:
+
+**x** - The two cases having the same key and timestamp are already merged.
+
+**\*** - The last two cases are really two distinct keys, since the timestamp is part of the Accumulo key. They are handled as two comparisons. They are different versions of the same key, but Rya is generally used with only the latest version.
+
+Here it is again, in full English:
+
+1.  If the parent statement exists, child doesn't, and parent time-stamp is before the start time, assume child statement was deleted, so delete the parent statement.
+
+2.  If the parent statement exists, child doesn't, and parent time-stamp is after the start time, assume parent statement was added, so do nothing.
+
+3.  If the child statement exists, parent doesn't, and child time-stamp is before the start time, assume parent statement was deleted, so do nothing.
+
+4.  If the child statement exists, parent doesn't, and child time-stamp is after the start time, assume child statement was added, so add the statement to the parent.
+
+## Instructions
+
+### Prerequisites
+
+If one or more of the Rya instance datastore uses Accumulo, then Accumulo and Hadoop will be needed. These can skipped if a Accumulo instance already exists for Rya to connect to. These directions are mostly for running on a local machine. Likewise, MongoDB can be setup to run locally as well but isn't necessary if connecting to an existing MongoDB instance.
+
+If you are building on Windows, you will need hadoop-common 2.6.0's *"winutils.exe"* and *"hadoop.dll"*. You can download it from [here](https://github.com/amihalik/hadoop-common-2.6.0-bin/archive/master.zip).
+
+Environment variables for `ACCUMULO_HOME` and `HADOOP_HOME` need to be set.
```

Comment:
This is established below, state that it is only required if one of the stores uses accumulo and on windows.  This is also unclear if its needed to setup an accumulo rya or the merge tool